os/drivers/audio/alc5658.c

```diff
@@ -616,12 +618,20 @@ static int alc5658_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR stru
 		switch (caps->ac_format.hw) {
 		case AUDIO_FU_VOLUME:
 			caps->ac_controls.hw[0] = ALC5658_HP_VOL_MAX;
+#ifndef CONFIG_AUDIO_EXCLUDE_VOLUME
```

Comment:
same as above.

Reply:
alc5658.c is modified so that this ioctl returns success though though exclude configs are set.